packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -1615,6 +1616,7 @@ class DropdownButtonFormField<T> extends FormField<T> {
                      menuMaxHeight: menuMaxHeight,
                      enableFeedback: enableFeedback,
                      alignment: alignment,
+                     borderRadius: borderRadius,
```

Comment:
Can you add a comment here just like this one:
https://github.com/flutter/flutter/blob/da9e87021d9641460018c69455366dd87b621d68/packages/flutter/lib/src/material/dropdown.dart#L892

That will help us stay consistent both ways in the future. :)